### PR TITLE
Added info about how table headers are populated

### LINKED ISSUE #7283: Document how table headers are populated
In the course of working on internationalization there has been knowledge learned about the various ways table headers are populated.

### PR DESCRIPTION
**Pull request by bisht-richa**
### Summary
Fixes #7283

Added some info in the sortable-table page about the table header

### How to test

- Start docusaurus
- Go to page http://localhost:3000/dashboard/code-base-works/sortable-table

It should have info about how table headers are populated

### CODE REVIEW

File: docusaurus/docs/code-base-works/sortable-table.md [hunk 1]
```diff
@@ -18,3 +18,13 @@ When the user scrolls the page and other columns become visible, Sortable Table
 
 Typically a delayed component would initally not render itself at all - or render a simple indicator that the content has not loaded. When its
 `startDelayedLoading` method is called, it will then render its full content, fetching data as needed etc,
+
+
+### Table header
+
+There are two ways table headers are populated:
+
+- Customize header: You can create a customized header for the sortable table by defining it in `explore.js`. Read more [how to create custom table headers](/code-base-works/customising-how-k8s-resources-are-presented#customising-tables) for each type.
```
Comment (aalves08) [inline docusaurus/docs/code-base-works/sortable-table.md:27]: I would change for something like:

Custom header: You can create a customized header for the sortable table by defining it in  the appropriate configuration file  under `shell/config/product` folder. Read more [how to create custom table headers](/code-base-works/customising-how-k8s-resources-are-presented#customising-tables) for each type.

**Review by aalves08 — COMMENTED** [review]
suggested a change @bisht-richa

File: docusaurus/docs/code-base-works/sortable-table.md (hunk 1 above)
Comment (bisht-richa) [reply docusaurus/docs/code-base-works/sortable-table.md]: Fixed

**Review by aalves08 — APPROVED** [review]
LGTM, although the link to `/code-base-works/customising-how-k8s-resources-are-presented#customising-tables` does not work for me... I didn't run docusarus so maybe that's the issue 🤔

**Pull request merged**